playbooks/templates/dev-install_net_config.yaml.j2

```diff
@@ -42,3 +42,13 @@ network_config:
     name: dummy1
     nm_controlled: true
 {% endif %}
+{% if sriov_enabled %}
+- type: sriov_pf
+  name: {{ sriov_interface | mandatory }}
```

Comment:
here, sriov_enabled = true could be inferred  from the presence of srivo_interface being defined.

Reply:
yeah I'll do that instead.